src/libsystemd-network/sd-dhcp6-lease.c

```diff
@@ -176,6 +176,37 @@ void sd_dhcp6_lease_reset_address_iter(sd_dhcp6_lease *lease) {
                 lease->addr_iter = lease->ia.addresses;
 }
 
+int sd_dhcp6_lease_get_pd_prefix(sd_dhcp6_lease *lease, struct in6_addr *prefix,
```

Comment:
Maybe call this `sd_dhcp6_lease_get_delegated_prefix`?

Reply:
Or just sd_dhcp6_lease_get_pd?